ensime_shared/protocol.py

```diff
@@ -213,11 +207,6 @@ def handle_string_response(self, call_id, payload):
             self.log.debug('EnDocUri %s', url)
             return url
 
-    def _format_source_file(self, newtext):
-        formatted = [line.encode('utf-8') for line in newtext.split('\n')]
-        # FIXME: should assure original buffer, not whatever is now current
-        self.editor.replace_buffer_contents(formatted)
```

Comment:
Please remove `replace_buffer_contents` from `Editor` class since this was its only usage and we try to avoid having dead code.